src/components/EditOrganization/index.tsx

```diff
@@ -53,17 +56,28 @@ export default function OrganizationView({ organization, reloadOrganizationData
 
   const saveOrganization = async () => {
     if (organizationRecord.name === '') {
-      setNameError('Required field.');
+      setNameError(strings.REQUIRED_FIELD);
+      return;
+    }
+
+    if (!organizationRecord.countryCode) {
+      setCountryError(strings.REQUIRED_FIELD);
+      return;
+    }
+
+    if (requireSubdivision && !organizationRecord.countrySubdivisionCode) {
+      setSubdivisionError(strings.REQUIRED_FIELD);
+      return;
+    }
```

Comment:
can we return after all the checks? maybe use a flag to capture hasErrors or something